core-plugins/src/test/java/co/cask/hydrator/plugin/batch/source/FileBatchSourceTest.java

```diff
@@ -463,4 +475,171 @@ public void testRecursiveTimeFilterRegex() throws Exception {
     Assert.assertTrue(outputValue.contains("Hello,World"));
     Assert.assertTrue(outputValue.contains("CDAP,Platform"));
   }
+
+  @Test
+  public void testFileBatchInputFormatText() throws Exception {
+    String filePath = "file:///tmp/test/test.txt";
```

Comment:
hardcoding a directory like this is bad practice, as it assumes the system has permission to write to that location, will not clean up after itself, and will break if some other test writes to the same directory. Use the TemporaryFolder variable in this class to create a new temporary folder just for this test case.